src/io/platform/f3xx/f302x8/I2Cf302x8.cpp

```diff
@@ -0,0 +1,150 @@
+#include <EVT/io/pin.hpp>
+#include <EVT/io/platform/f3xx/f302x8/I2Cf302x8.hpp>
+
+namespace EVT::core::IO {
+
+    /**
```

Comment:
Same issue here with premature indents

Reply:
I've become too reliant on my out-indent it would seem